src/components/CommentList.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Comments from './Comment';
+
+function CommentsList({ comments }) {
+  return (
+    <div className="post__comments-block">
+      {comments.map(comment => <Comments comment={comment} key={comment.id} />)}
```

Comment:
Почему Comments, если ты передаешь информацию об одном комментарии?
```suggestion
      {comments.map(comment => <Comment comment={comment} key={comment.id} />)}
```
А в таком случае уже 4(четыре) слова comment подряд